core/primitives/block.hpp

```diff
@@ -14,28 +14,9 @@ namespace kagome::primitives {
   /**
    * @brief Block class represents block
```

Comment:
Not very informative comment. Maybe a bit more on what a block is here?

Reply:
corrected